uber/templates/groups/form.html

```diff
@@ -36,6 +36,18 @@
             $.field("status").find("[value={{ c.UNAPPROVED }}]").text("n/a (not a paid Dealer)");
         {% endif %}
     });
+
+    {% if group.is_dealer %}
+    var showOrHideCategoriesText = function() {
+            setVisible('#category-other', $(":checkbox[name=categories][value={{ c.OTHER }}]").prop("checked"));
+        };
+
+        $(function () {
+            $(":checkbox[name=categories][value={{ c.OTHER }}]").parents("label").append('<input id="category-other" class="form-control" type="text" name="categories_text" value="{{ group.categories_text }}" />');
```

Comment:
Same deal here with the dynamic generation of HTML. Rendered inline is probably fine.